rllib/algorithms/algorithm.py

```diff
@@ -3267,19 +3053,23 @@ def _run_one_training_iteration_and_evaluation_in_parallel(
         evaluation step takes roughly the same time as the training iteration.
 
         Returns:
-            The accumulated training and evaluation results.
+            A tuple containing the training results, the evaluation results, and
+            the `TrainIterCtx` object returned by the training call.
         """
         with concurrent.futures.ThreadPoolExecutor() as executor:
```

Comment:
Is the thread then calling the learner workers for training? 

Also: Let's give it a prefix such that we can debug it better.